static/src/javascripts/projects/facia/modules/onwards/weather.js

```diff
@@ -198,6 +198,11 @@ define([
 
         attachToDOM: function (tmpl, city) {
             $holder = $('#headlines .js-container__header');
+
+            if (!!config.switches.attachWeatherToTopContainer) {
+                $holder = $('.js-container__header')[0];
```

Comment:
I'm a little surprised that this is working, `[0]` should give you the actual DOM element, which does not have an `.append` method.

But there must be something else that I missed.